### ISSUE TITLE
contrast of textfield too low

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#15219]
![image](https://user-images.githubusercontent.com/1227763/37062753-3fe7b148-214c-11e8-82d3-259e4ca4c175.png)

- [ ] Insufficient contrast: grey "underlined" text against grey background is poor 
- [ ] helper text is too light

**Comment by goderbauer** [comment]
We are lighter then the spec here. The values in the sticker sheet give us more contrast:

Helper: grey 700 with 87% opacity
Background: grey 600 with 12% opacity
Line: grey 600
Text: grey 600

That gives us a contrast of 4.5:1 for helper text to its background. However, the contrast between background in the text field and the text "Underlined" is still too low with just 3.3:1.

/cc @HansMuller

**Comment by goderbauer** [comment]
From b/74352810 and the updated sticker sheet:

Helper and hint text in the text field should be: Grey700
Background should be: White (for outlined) or Grey50 (for with background)
Text entered into the text field should be: Grey800

**Comment by HansMuller** [comment]
Should the counter text also be Grey700? I'm going to assume yes.

**Comment by goderbauer** [comment]
I was told that "all hinted text" should have that color. I'd say that counter is also a hint.

**Comment by HansMuller** [comment]
What color should hinted text have for the dark theme?
